package-lock.json

```diff
@@ -20347,6 +20347,18 @@
         }
       }
     },
+    "Tryouts/Plugins/ApplicationPlugins/process explorer/node_modules/@angular-devkit/build-angular/node_modules/@babel/runtime": {
+      "version": "7.20.13",
```

Comment:
Anyone know why the overall upgrade for the pr isn't applied here?  This is the old version

Reply:
I don't know. If it was the missing ^ than it would be a problem on line 20190 too in my opinion.
What's more troubling is that dependabot keeps updating the JavaScript projects but not the .NET ones.
(e.g. WebView2 has a new version)

Reply:
> >Does dependabot enforce the upgrade of transitive dependencies too?

I don't believe it does.  In order to force transitive (assuming they are compatible) you need to add overrides section in the package.json itself.

Reply:
Thank you, that's good to know.
I'm merging this PR shortly.